src/universal/services/ono/queries/pullrequests/index.js

```diff
@@ -112,18 +105,30 @@ export type RepositoryGraphType = {
   name: string
 }
 
+export const PullRequestSource = {
+  BRANCH: 'branch',
+  REV: 'rev',
```

Comment:
The current implementation of target filter in GraphQL allows passing all of them, values are passing as strings and are not validated against graphene enum `PullRequestSourceType`. That's why it's not an error now. I'll add note to ono PR.